src/formats.ts

```diff
@@ -197,7 +197,7 @@ function compareIsoTime(t1: string, t2: string): number | undefined {
   return 0
 }
 
-const DATE_TIME_SEPARATOR = /t|\s/i
+const DATE_TIME_SEPARATOR = /t/i
```

Comment:
@MarcoKorinth This change might break the iso-date-time comparison (see line 219).

Reply:
Yes it did. I fixed it in the newest commit.